Gopkg.toml

```diff
@@ -77,11 +77,11 @@ required = [
 
 [[constraint]]
   name = "github.com/prometheus/client_golang"
-  version = "v1.0.0"
+  version = "=v1.1.0"
```

Comment:
Did this almost subconsciously - since I tested this together with `dep update` in planet (which did bump these packages), I thought it would not hurt to update here.